telegram/constants.py

```diff
@@ -52,3 +52,42 @@
 MAX_MESSAGES_PER_MINUTE_PER_GROUP = 20
 MAX_MESSAGE_ENTITIES = 100
 MAX_INLINE_QUERY_RESULTS = 50
+
+
+class MediaCategory:
+    """This are the mime categories of media you can send via telegram. 
+    
+       They are used in `Telegram.ext.filters.Filters.document.`"""
+    application = "application/"
+    audio = "audio/"
+    image = "image/"
+    video = "video/"
+    text = "text/"
+
+
+class Document:
```

Comment:
Is there a good reason you're not using the [stdlib mimetype](https://docs.python.org/3.2/library/mimetypes.html)?

Reply:
No, just didn't found them. I will fix this.

Reply:
The stdlib mimetype doesn't work for many mimetypes in telegram. ".exe" documents for example have  the mimetype "application/x-ms-dos-executable" if you send it with an official telegram app, but ".exe" is matched to "application/octet-stream" in the stdlib mimetype.